week2/Task6/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿// 6. Let's get things done
+
+var signaler = new Signaler();
+signaler.AddTime(new JupiterTime(1, 20));
+signaler.AddTime(new JupiterTime(2, 20));
+signaler.AddTime(new JupiterTime(3, 20));
+signaler.Inform();
+
+public class Signaler
+{
+    List<JupiterTime> Timelist = new List<JupiterTime>();
+    public void AddTime(JupiterTime time){
+         Timelist.Add(time);
+    }
+
+    public void Inform()
+    {
+        if (Timelist == null) Console.WriteLine("No timers added yet");
```

Comment:
This part will never be null because it is always created as `List<JupiterTime> Timelist = new List<JupiterTime>();` when the class is created. A way to check if any timers are added would be to check the count, Timelist.Count > 0